source/ship/ShipAICache.cpp

```diff
@@ -97,7 +97,7 @@ void ShipAICache::Calibrate(const Ship &ship)
 		// and other ships with exceptionally long range weapons such as detainers
 		// The AI shouldn't use the artillery AI if it has no reverse and it's turning
 		// capabilities are very bad. Otherwise it spends most of it's time flying around.
-		useArtilleryAI = (artilleryDPS > totalDPS / 2.
+		useArtilleryAI = (artilleryDPS > totalDPS * .75
```

Comment:
```suggestion
		useArtilleryAI = (artilleryDPS > totalDPS * .5
```

Shouldn't this be .5, so that it checks to see if the artillery DPS is greater than 50% of the DPS? Then it'd go for artillery if over half it's DPS is artillery, and only close if ithe short range DPS is greater?

Reply:
if we make it .5 nothing would change, what warp wants is that artillery damage is greater than three quarters of the total dps so that carriers etc. go into melee

Reply:
Ah, cruiser. Yes, that should generally be more of a brawler. I think some variants are more artilleryish, but yeah.

Does the Artillery AI have any kind of "can I get clear of my enemy" checks for mobility and the like? This might be a situation where it's acting badly right now, but it'll act better once it gets a more reasonable mass and lack of fighter-like agility.

Reply:
mobility calculations are planned but right now it just compares max speeds.
when I will start working on the AI after 0.10.0 is out, this will be one of the first things I will focus on.

Reply:
It already changes AI when running out of ammo

Reply:
It also already wont artillery AI if it doesnt have a minimum amount of agility, but this `(ship.MaxReverseVelocity() || turningRadius < 0.2 * shortestArtillery)` is probably not strict enough.
Also it doesnt account for speed modifiers yet.

Reply:
@RisingLeaf we could also consider the role of a ship in a fleet dynamically.
If it is indeed the tank, then give it tank personality automatically.
If its alone let it just missile boat.
That said we should prob make fleets merge with one another and separate when it makes sense too